packages/ai-ide/src/common/architect-prompt-template.ts

```diff
@@ -12,10 +12,12 @@ import { PromptTemplate } from '@theia/ai-core/lib/common';
 import { GET_WORKSPACE_FILE_LIST_FUNCTION_ID, FILE_CONTENT_FUNCTION_ID, GET_WORKSPACE_DIRECTORY_STRUCTURE_FUNCTION_ID } from './workspace-functions';
 import { CONTEXT_FILES_VARIABLE_ID } from './context-variables';
 
+export const ARCHITECT_TASK_SUMMARY_PROMPT_TEMPLATE_ID = 'architect-task-summary';
+
 export const architectPromptTemplate = <PromptTemplate>{
-   id: 'architect-system',
-   template: `{{!-- This prompt is licensed under the MIT License (https://opensource.org/license/mit).
-Made improvements or adaptations to this prompt template? We’d love for you to share it with the community! Contribute back here:
+    id: 'architect-system',
+    template: `{{!-- This prompt is licensed under the MIT License (https://opensource.org/license/mit).
+Made improvements or adaptations to this prompt template? Wed love for you to share it with the community! Contribute back here:
```

Comment:
This change should be reverted

```suggestion
Made improvements or adaptations to this prompt template? We’d love for you to share it with the community! Contribute back here:
```